packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -83,6 +84,9 @@ export class Viewer implements IDisposable {
         this._camera.attachControl();
         this._reframeCamera(); // set default camera values
 
+        // Load a default light, but ignore errors as the user might be immediately loading their own environment.
+        this.loadEnvironmentAsync(undefined).catch(() => {});

```

Comment:
This just got lost in a previous refactor, which means there is no light or env by default if no env is specified.